README.md

```diff
@@ -64,31 +63,25 @@ experience and additionally provides testing and bug-diagnosing capabilities.
 
 Submariner provides the `subctl` CLI utility to simplify the deployment and maintenance of Submariner across your clusters.
 
-See the [`subctl` docs on Submariner's website](https://submariner.io/operations/deployment/subctl/).
+See the [`subctl` Deployment docs](https://submariner.io/operations/deployment/subctl/) on Submariner's website.
 
 ### Installation using Helm
 
-See the [Helm](https://submariner.io/operations/deployment/helm/) section on Submariner's website.
+See the [Helm Deployment docs](https://submariner.io/operations/deployment/helm/) on Submariner's website.
 
 ### Validate Submariner is Working
 
-See the [`subctl verify` docs on Submariner's website](https://submariner.io/operations/deployment/subctl/#verify).
+See the [`subctl verify` docs](https://submariner.io/operations/deployment/subctl/#verify) and [Automated
+Troubleshooting docs](https://submariner.io/operations/troubleshooting/#automated-troubleshooting) on Submariner's website
 
 ## Building and Testing
 
-See the [Building and Testing docs on Submariner's website](https://submariner.io/development/building-testing/).
+See the [Building and Testing docs](https://submariner.io/development/building-testing/) on Submariner's website.
 
-## Known Issues/Notes
+## Known Issues
 
-### OpenShift Notes
-
-When running in OpenShift, Submariner needs to grant the appropriate security context for the service accounts (SAs):
-
-   ```shell
-   oc adm policy add-scc-to-user privileged system:serviceaccount:submariner:submariner-routeagent
-   oc adm policy add-scc-to-user privileged system:serviceaccount:submariner:submariner-gateway
-   ```
+See the [Known Issues docs](https://submariner.io/operations/known-issues/) on Submariner's website.
```

Comment:
The SCC issue mentioned above (lines 85-90) is not documented under https://submariner.io/operations/known-issues/. Let's add it there before we remove it from this README. Also, AFAICT, we already take care of this with `subctl` as well as `submariner-addon` driven deployments, so I _think_ that users only need to do this for Helm. @skitt can you confirm?

Reply:
That’s correct, we only need this for Helm (and we could take care of it there too).

Reply:
https://github.com/submariner-io/submariner-website/pull/640

Reply:
https://github.com/submariner-io/submariner-charts/issues/195